tests/e2e/utils/use-incognito.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { test as Test } from '@wordpress/e2e-test-utils-playwright';
+
+const getFreshSession = () => {
+	return {
+		cookies: [],
+		origins: [],
+	};
+};
+
+export const useIncognito = ( test: typeof Test ) => {
+	test.use( {
+		storageState: getFreshSession(),
+	} );
+};
```

Comment:
I don't think this abstraction is really necessary. We're adding functions for something that can be done in a single line anyway.

`test. use( storageState: { cookies: [], origins: [] } );`

Also, the naming implies it's opening an incognito window which may confuse folks in the future, and also since it's using `test.use` it can only be executed at the `describe` level, not within a test itself, so it may cause further confusion.

I think we should drop this.

Moreover, in https://github.com/woocommerce/woocommerce-blocks/pull/10561 we are adding auth states, so we can instead do: `test.use( { storageState: guestFile } );` to get a guest session.

Reply:
Hmm, @dinhtungdu looks like we're not going to get #10561 merged today it seems there's some more work to do on it, so I'd be fine with you skipping waiting for that and just getting it working.

Reply:
You're right. I removed the utility and added comment explaining the reason.

About the using the guest file, I think it will prevent tests run in isolation because we're reusing the context. That's why I use an empty object for the state (and was over optimized by using a function to ensure the new object every time). I will verify this again when the @tarhi-saad PR is merged.

As mentioned in the code comment, I think tests should run in isolation by default, so we shouldn't use the `storageState` file in the main config. We can revisit this topic later when tests can be run in isolation are ready for isolation.